react-graphql-client/scripts/build.js

```diff
@@ -72,13 +72,9 @@ function build(previousFileSizes) {
    }
 
     console.log(chalk.green('Compiled successfully.'));
-    console.log();
```

Comment:
I'd rather leave this file as is. These extra `console.log()`s were to display extra new lines for formatting purposes I assume. The query changes below look useful on their own 👍